### ISSUE TITLE
JSX output is broken

### ISSUE THREAD

**Issue by milahu** [coderaiser/putout#118]
actual: wrong indent, wrong whitespace

```jsx
function App(props) {
  return (
    (<div class="app">hello{props.name}
    </div>)
  );
}
```

expected

```jsx
function App(props) {
  return (
    <div class="app">
      hello {props.name}
    </div>
  );
}
```

upstream issue https://github.com/benjamn/recast/issues/211

see also #116

**Comment by coderaiser** [comment]
https://putout.cloudcmd.io/#/gist/18db3a272e5e48d482be82e85ad2a215/dd7ba82d3681737cb9f46a6ea83113c5781ef807

**Comment by coderaiser** [comment]
Fixed with https://github.com/benjamn/recast/pull/1223. Landed in [`@putout/recast v1.10.0`](https://github.com/putoutjs/recast/releases/tag/v1.10.0) 🎉 .

Please reinstall 🐊**Putout**. Is it works for you?
